modules/tgw/cross-region-hub-connector/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "region" {
+  type        = string
+  description = "AWS Region"
+}
+
+variable "this_region" {
```

Comment:
`this_` and `home_` were a bit confusing to me before and still confusing.

Could we simplify this before upstreaming ?

Reply:
I think `this` could be `current` but IMO `this` makes the most sense, home could be maybe `default`